src/explore-education-statistics-admin/src/modules/find-statistics/components/ReleaseContentAccordionSection.tsx

```diff
@@ -130,6 +132,18 @@ const ReleaseContentAccordionSection = ({
     [release.id, removeContentSection, sectionId],
   );
 
+  const updateBlockCommentsInAccordionSection: CommentsChangeHandler = useCallback(
```

Comment:
I'll change it. In my head, I thought I'd keep the name because there is also a `updateBlockCommentsInSummarySection` and `updateBlockCommentsInHeadlineSection`. I guess it is kind of a fine point, but by having the name explicit it indicates that other methods exist to handle comments for other parts of the manage content page. The bug to do with UI state not updating when adding/updating/removing comments seems to have happened precisely because the summary section and headline section comments had been forgotten about.